bacula/baculum/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:20.04
+
+MAINTAINER ops@marfeel.com
+
+RUN apt-get update && apt-get install -y gnupg wget joe ca-certificates
+RUN wget -qO - http://bacula.org/downloads/baculum/baculum.pub | apt-key add -
+
+COPY baculum.list /etc/apt/sources.list.d/baculum.list
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y gnupg wget baculum-common baculum-web baculum-web-apache2 baculum-api baculum-api-apache2
```

Comment:
Here we can have a cosmetic change to improve maintainability

```dockerfile
RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
    gnupg \
    wget \
    baculum-common \
    baculum-web \
    baculum-web-apache2 \
    baculum-api \
    baculum-api-apache2
```

Reply:
Also, just realized it is missing the `--no-install-recommends` flag, just updated the comment above.